src/staking/MSpellStaking.sol

```diff
@@ -217,4 +228,28 @@ contract MSpellStaking is BoringOwnable {
     function toggleLockUp(bool status) external onlyOwner {
         toggleLockup = status;
     }
+
+    ///////////////////////////////////////////////////////////////////
+    // Voting
+    ///////////////////////////////////////////////////////////////////
+
+    function transfer(address, uint256) public virtual override returns (bool) {
+        revert ErrUnsupportedOperation();
+    }
+
+    function transferFrom(address, address, uint256) public virtual override returns (bool) {
+        revert ErrUnsupportedOperation();
+    }
+
+    function approve(address, uint256) public virtual override returns (bool) {
+        revert ErrUnsupportedOperation();
+    }
+
+    function _update(address from, address to, uint256 amount) internal override(ERC20, ERC20Votes) {
+        super._update(from, to, amount);
```

Comment:
why is this necessary if we do the same

Reply:
It's the way the oz governance contracts are designed by default and provided this way by oz defender wizard when you are generating the contract. If you remove the functions it's making it ambiguous.

```
Derived contract must override function "_cancel". Two or more base classes define function with same name and parameter types.
```